habitat-lab/habitat/tasks/rearrange/utils.py

```diff
@@ -94,7 +94,7 @@ def __add__(self, other):
 
 
 def general_sim_collision(
-    sim: habitat_sim.Simulator, agent_embodiment: MobileManipulator
+        sim: habitat_sim.Simulator, agent_embodiment: MobileManipulator, ignore_names:  Optional[List[str]] = None
```

Comment:
`ignore_names` is a list of strings and appears to be requesting object handles.

Instead, we supply `object_id` integers.

I would rename this parameter as such: `ignored_object_ids : Optional[List[int]]`.